src/App.tsx

```diff
@@ -26,70 +26,122 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPHABET: {
+      visibleGoods.sort((first, second) => {
+        return first.localeCompare(second);
+      });
+      break;
+    }
 
-  return visibleGoods;
+    case SortType.LENGTH: {
+      visibleGoods.sort((first, second) => {
+        return first.length - second.length;
+      });
+      break;
+    }
```

Comment:
You could also put whole switch case inside `sort` instead of doing it separately in each case 🤔